pkg/manager/runtime_env/db_test.go

```diff
@@ -348,3 +349,256 @@ func TestServer_deleteRuntimeEnvLabels_byCount(t *testing.T) {
 	}
 
 }
+
+func TestServer_getAttachmentsByRuntimeEnvIds(t *testing.T) {
+	attachments, err := p.getAttachmentsByRuntimeEnvIds([]string{})
```

Comment:
`getAttachmentsByRuntimeEnvIds` should change to `getAttachedCredentialsByRuntimeEnvIds`, `Attachments` is ambiguity.